### ISSUE TITLE
add helm chart for qualys agent

### ISSUE THREAD

**Issue by fawaf** [Qualys/Qualys-Helm-Charts#1]
note: this is different from qcs-sensor

**Comment by MPV** [comment]
Indeed, I have only found this one (by @snicholson-qualys and @rapatwardhan) so far, and it's not quite as elaborate as a helm chart:
- https://github.com/Qualys/cloud-agent-k8s-daemonset

**Comment by fawaf** [comment]
yeah, we ended up rolling our own internally.
